frontend/app/components/wp-resizer/wp-resizer.directive.ts

```diff
@@ -66,6 +69,12 @@ export class WorkPackageResizerController {
 
     // Disable mouse move
     window.removeEventListener('mousemove', this.mouseMoveHandler);
+
+    // Take care at the end that the elemntFlex-Value is the same as the acutal value
+    // When the mouseup is outside the container these values will differ
+    // which will cause problems at the next movement start
+    let localStorageValue = localStorage.getItem("detailsSideFlexBasis");
```

Comment:
You might want to `parseInt(localStorage.getItem..)` here, and check for `NaN` to avoid invalid values when the local storage value has a non-integer value.